### PR TITLE
Changelog for 0.9.0

### PR DESCRIPTION
**Pull request by esantorella**
## Motivation

Adding to the changelog so we can put out a new release

### Have you read the [Contributing Guidelines on pull requests](https://github.com/pytorch/botorch/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

None really

### CODE REVIEW

**Review by Balandat — COMMENTED** [review]
Given that this is a compatibility bumping release, should we also just require pytorch 1.13.1 while we're at it?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,49 @@
...
+#### Compatibility
+* Require Python >= 3.9.0 (#1924).
+
+#### Highlights
+* Introduce `OrthogonalAdditiveKernel` (#1869).
+* Speed up LCE-A kernel by over an order of magnitude (#1910).
+* Introduce `optimize_acqf_homotopy`, for optimizing acquisition functions with homotopy (#1915).
+* Introduce `PriorGuidedAcquisitionFunction` (PiBO) (#1920).
+* Introduce `qLogExpectedImprovement`, which provides more accurate numerics 
+than `qExpectedImprovement` and can lead to significant optimization improvements (#1936).
```
Comment (esantorella) [inline CHANGELOG.md:16]:
```suggestion
* Introduce `qLogExpectedImprovement`, which provides more accurate numerics than `qExpectedImprovement` and can lead to significant optimization improvements (#1936).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,49 @@
...
+
+#### Bug Fixes
+* Enable pathwise sampling modules to be converted to GPU  (#1821).
+* Allow `Standardize` modules to be loaded once trained (#1874).
+* Fix memory leak in Inducing Point Allocators (#1890).
+* Correct einsum computation in `LCEAKernel` (#1918).
+* Properly whiten bounds in MVNXPB (#1933).
+* Make `FixedFeatureAcquisitionFunction` convert floats to double-precision tensors rather than single-precision (#1944).
+* Fix memory leak in `FullyBayesianPosterior` (#1951).
+* Make `AnalyticExpectedUtilityOfBestOption` input constructor work correctionly with multi-task GPs (#1955).
```
Comment (esantorella) [inline CHANGELOG.md:46]:
```suggestion
* Make `AnalyticExpectedUtilityOfBestOption` input constructor work correctionly with multi-task GPs (#1955).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,48 @@
 
 The release log for BoTorch.
 
+## [0.9.0] - July 31, 2023
+
+#### Compatibility
+* Require Python >= 3.9.0 (#1924).
```
Comment (esantorella) [inline CHANGELOG.md:8]:
```suggestion
* Require Python >= 3.9.0 (#1924).
* Require PyTorch >= 1.13.1 (#1960).
* Require linear_operator == 0.5.0 (#1961).
* Require GPyTorch == 1.11 (#1961).
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,48 @@
...
+* Add constrained synthetic test functions `PressureVesselDesign`, `WeldedBeam`, `SpeedReducer`, and `TensionCompressionString` (#1832).
+* Support decoupled fantasization (#1853) and decoupled evaluations in cost-aware utilities (#1949).
+* Add `PairwiseBayesianActiveLearningByDisagreement`, an active learning acquisition function for PBO and BOPE (#1855).
+* Support custom mean and likelihood in `MultiTaskGP` (#1909).
+* Enable candidate generation (via `optimize_acqf`) with both `non_linear_constraints` and `fixed_features` (#1912).
+* Introduce `L0PenaltyApproxObjective` to support L0 regularization (#1916).
+* Enable batching in `PriorGuidedAcquisitionFunction` (#1925).
+
+#### Other changes
+* Deprecate `FixedNoiseMultiTaskGP`; allow `trainY_var` optionally in `MultiTaskGP` (#1818).
```
Comment (esantorella) [inline CHANGELOG.md:28]:
```suggestion
* Deprecate `FixedNoiseMultiTaskGP`; allow `train_Yvar` optionally in `MultiTaskGP` (#1818).
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,48 @@
...
+#### Bug Fixes
+* Enable pathwise sampling modules to be converted to GPU  (#1821).
+* Allow `Standardize` modules to be loaded once trained (#1874).
+* Fix memory leak in Inducing Point Allocators (#1890).
+* Correct einsum computation in `LCEAKernel` (#1918).
+* Properly whiten bounds in MVNXPB (#1933).
+* Make `FixedFeatureAcquisitionFunction` convert floats to double-precision tensors rather than single-precision (#1944).
+* Fix memory leak in `FullyBayesianPosterior` (#1951).
+* Make `AnalyticExpectedUtilityOfBestOption` input constructor work correctionly with multi-task GPs (#1955).
+
```
Comment (esantorella) [inline CHANGELOG.md:46]:
```suggestion


```

**Pull request closed without merging**